var/spack/repos/builtin/packages/cardioid/package.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```

Comment:
please remove these FIXME instructions and only leave the license header :)

Reply:
Done!